frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -230,7 +230,10 @@ function FileManagerMenu:setUpdateItemTable()
     self.menu_items.cloud_storage = {
         text = _("Cloud storage"),
         callback = function()
-            local cloud_storage = CloudStorage:new{}
+            local cloud_storage = CloudStorage:new{
+                width = Screen:getWidth(),
```

Comment:
I'm slightly confused by this. Doesn't CloudStorage do that all by itself already?